app/src/components/integrations/containers/integrationSettingsContainer/integrationSettingsContainer.jsx

```diff
@@ -70,9 +67,9 @@ export class IntegrationSettingsContainer extends Component {
       isGlobal,
       id,
       pluginName,
-      () => {
+      (normalizeData) => {
```

Comment:
`normalizeData` sounds like an action, may be name it just `updatedParameters` to use property shorthand?